tests/rules/no-nested-subscribe.ts

```diff
@@ -0,0 +1,202 @@
+/**
+ * @license Use of this source code is governed by an MIT-style license that
+ * can be found in the LICENSE file at https://github.com/cartant/eslint-plugin-rxjs
+ */
+
+import { stripIndent } from "common-tags";
+import rule = require("../../source/rules/no-nested-subscribe");
+import { ruleTester } from "../utils";
+
+ruleTester({ types: true }).run("no-nested-subscribe", rule, {
+  valid: [
+    `
```

Comment:
I think we should always use `stripIndent` for multi-line `valid` snippets, too.